pyfakefs/fake_filesystem.py

```diff
@@ -1427,7 +1429,10 @@ def CreateLink(self, file_path, link_target):
 
     Raises:
       IOError:  if the file already exists
+      OSError:  if on Windows before Python 3.2
     """
+    if not _is_link_supported:
+        raise OSError("Symbolic links are not supported on Windows before Python 3.2")
     resolved_file_path = self.ResolvePath(file_path)
```

Comment:
Use 2 spaces like in the rest of the file (or change it all to 4 spaces to follow pep-8, which I would really like!)